plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java

```diff
@@ -168,7 +168,7 @@ public static boolean isTrinoView(String tableType, Map<String, String> tablePar
         // https://github.com/trinodb/trino/blame/38bd0dff736024f3ae01dbbe7d1db5bd1d50c43e/presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java#L902.
         return tableType.equals(VIRTUAL_VIEW.name()) &&
                 "true".equals(tableParameters.get(PRESTO_VIEW_FLAG)) &&
-                tableParameters.get(TABLE_COMMENT).equalsIgnoreCase(PRESTO_VIEW_COMMENT);
+                PRESTO_VIEW_COMMENT.equalsIgnoreCase(tableParameters.get(TABLE_COMMENT));
```

Comment:
The stacktrace of #18799 mentions `isTrinoMaterializedView ` method, not `isTrinoView`